starter-code/public/admin.html

```diff
@@ -51,6 +55,7 @@
     <script src="/scripts/articleView.js"></script>
     <script>
       // TODO: After we fetch and load all the articles, what view function do we want to run next?
+      Article.fetchAll(articleView.initAdmitPage);
       // Let's pass in a callback function, to determine what happens AFTER all articles are loaded.
       Article.fetchAll();
```

Comment:
So you'll want to get rid of this second fetch all. Basically increasing your compile time since you are invoking the .fetchAll() function twice.